### PR TITLE
CI: use namespace.io

### PR DESCRIPTION
**Pull request by jnewb1**
fixed the multiarch push, working on my fork: https://github.com/jnewb1/openpilot/actions/runs/7580004107

### CODE REVIEW

File: .github/workflows/selfdrive_tests.yaml [hunk 1]
```diff
@@ -181,7 +180,7 @@ jobs:
       run: |
         ${{ env.RUN }} "source selfdrive/test/setup_xvfb.sh && \
                         export MAPBOX_TOKEN='pk.eyJ1Ijoiam5ld2IiLCJhIjoiY2xxNW8zZXprMGw1ZzJwbzZneHd2NHljbSJ9.gV7VPRfbXFetD-1OVF0XZg' && \
-                        $PYTEST --timeout 40 -m 'not slow' && \
+                        $PYTEST --timeout 40 -m 'not slow' -n $(nproc) && \
```
Comment (sshane) [inline .github/workflows/selfdrive_tests.yaml:183]: why was this changed from auto only here?

Comment (jnewb1) [reply .github/workflows/selfdrive_tests.yaml:183]: the pytest auto only returns 4 even though there are 8 cpus (perhaps something with hyperthreading or docker), im going to bring it up with the namespace people.

Comment (sshane) [reply .github/workflows/selfdrive_tests.yaml:183]: `-n auto` returns physical cores, maybe there's just 4 hyperthreaded cores. (using logical cores still improves performance, so maybe we should switch to `-n logical`)
